reflex/app.py

```diff
@@ -891,7 +924,17 @@ def router(self) -> Callable[[str], str | None]:
         """
         from reflex.route import get_router
 
-        return get_router(list(dict.fromkeys([*self._unevaluated_pages, *self._pages])))
+        routes = list(dict.fromkeys([*self._unevaluated_pages, *self._pages]))
```

Comment:
pretty sure this isn't doing anything either... other than just emitting mildly useful debugging info

Reply:
i agree, it works without this.  you can leave or remove.